.github/workflows/integration-tests.yaml

```diff
@@ -122,9 +122,6 @@ jobs:
           build-args: |
             DEBIAN_VERSION=12.9
           from-scratch: ${{ matrix.test.from-scratch || 'false' }}
-          cache-mount-ids: |
```

Comment:
Does this mean that the example on the README file has to change as well?

Reply:
It doesn't have to change but we should discourage it's use at this point